diff --git a/panels/background/bg-wallpapers-source.c b/panels/background/bg-wallpapers-source.c
index 7d3b644e5..6cd9bd5dc 100644
--- a/panels/background/bg-wallpapers-source.c
+++ b/panels/background/bg-wallpapers-source.c
@@ -82,7 +82,7 @@ load_default_bg (BgWallpapersSource *self)
 
     filename = g_build_filename (system_data_dirs[i],
 				 "gnome-background-properties",
-				 "adwaita.xml",
+				 "blankon-wallpapers.xml",
 				 NULL);
     if (cc_background_xml_load_xml (self->xml, filename))
       break;
diff --git a/panels/info-overview/cc-info-overview-panel.ui b/panels/info-overview/cc-info-overview-panel.ui
index 98cb85df3..192b2ef6a 100644
--- a/panels/info-overview/cc-info-overview-panel.ui
+++ b/panels/info-overview/cc-info-overview-panel.ui
@@ -28,7 +28,7 @@
                   <object class="GtkImage">
                     <property name="visible">True</property>
                     <property name="can_focus">False</property>
-                    <property name="resource">/org/gnome/control-center/info-overview/GnomeLogoVerticalMedium.svg</property>
+                    <property name="resource">/org/gnome/control-center/info-overview/BlankOn.png</property>
                   </object>
                   <packing>
                     <property name="expand">False</property>
diff --git a/panels/info-overview/info-overview.gresource.xml b/panels/info-overview/info-overview.gresource.xml
index 83806e0ad..e827aa12b 100644
--- a/panels/info-overview/info-overview.gresource.xml
+++ b/panels/info-overview/info-overview.gresource.xml
@@ -2,6 +2,6 @@
 <gresources>
   <gresource prefix="/org/gnome/control-center/info-overview">
     <file preprocess="xml-stripblanks">cc-info-overview-panel.ui</file>
-    <file>GnomeLogoVerticalMedium.svg</file>
+    <file>BlankOn.png</file>
   </gresource>
 </gresources>
